test/run_test.py

```diff
@@ -182,9 +182,7 @@ def __contains__(self, item):
     "test_autograd",
 ]
 
-XPU_TEST = [
-    "test_xpu",
-]
+XPU_TEST = ["test_xpu", "xpu/test_tensor_creation_ops_xpu"]
```

Comment:
Nit (can we keep a multiline definition here)
```suggestion
XPU_TEST = [
        "test_xpu",
        "xpu/test_tensor_creation_ops_xpu",
        ]
```

Reply:
`lintrunner` will automatically format the style. It will transfer the multiline to a single line if the number of characters of a line does not exceed the threshold. I will try to keep the multiline and check for any lint issue.